pocs/camera/camera.py

```diff
@@ -17,6 +19,9 @@ def __init__(self,
                  model='simulator',
                  port=None,
                  primary=False,
+                 set_point=None,
```

Comment:
Yeah, I think that makes sense. Make sure it has a simple descriptive docstring though as I think people dealing primarily with DSLRs and PANOPTES won't necessarily know what a set point is and might think they need to implement it.